templates/crds/addressspaces.crd.yaml

```diff
@@ -367,6 +367,22 @@ spec:
             caCert:
               description: "CA certificate for endpoints."
               type: string
+            routers:
+              type: array
+              description: "Router status for this address space."
+              items:
+                type: object
+                required:
+                  - id
+                properties:
+                  id:
+                    type: string
+                  undelivered:
+                    type: integer
+                  neighbours:
```

Comment:
You are using the British English spelling of the word 'neighbours'.  I think the project tends to favo(u)r America spellings, so I think this ought to be switch to neighbor.